dev-libs/libpreludedb/libpreludedb-3.0.0.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{3,4,5}} )
+
+inherit autotools eutils python-r1
+
+DESCRIPTION="Framework to easy access to the Prelude database"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/3.0.0/${PF}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc python mysql postgres sqlite"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="dev-libs/libgcrypt:0=
+	net-libs/gnutls
+	dev-libs/libprelude
+	python? ( ${PYTHON_DEPS} )
+	mysql? ( virtual/libmysqlclient )
+	postgres? ( dev-db/postgresql:* )
+	sqlite? ( dev-db/sqlite )"
```

Comment:
I know we only have sqlite in the tree but can you use a slot dependency specifier with this dep? i.e. sqlite:3